kubetest/kubernetes.go

```diff
@@ -106,6 +106,25 @@ func countReadyNodes(nodes *nodeList) int {
 	return len(ready)
 }
 
+func kubectlGetPods(labelSelector []string) (*podList, error) {
```

Comment:
could we also make the namespace as a parameter?

Reply:
done 👍